examples/marketdataV2.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * This examples shows how to use tha alpaca data v2 rest functions during some
+ * simple examples. Don't forget to install the dependencies before running it.
+ * The results are logged to the console in a readable tabled form.
+ * This file was written in Typescript, to run it use eg. the ts-node command.
+ */
+
+import Alpaca from "@alpacahq/alpaca-trade-api";
+import * as _ from "lodash";
+import moment from "moment";
+
+class Example {
+  async run() {
+    // Create your alpaca object by setting your credentials
+    // There are more configurable options there, e.g.
+    // URLs and more
+    const alpaca = new Alpaca({
+      keyId: "API_KEY",
+      secretKey: "API_SECRET",
+    });
+
+    // Get all the trades between a start and end time. The paging was implemented
+    // in the SDK
+    const start = "2022-03-10T00:00:00Z";
+    const end = "2022-03-10T23:59:59Z";
+    let symbol = "TMF";
+    const trades = alpaca.getTradesV2(symbol, { start, end });
+    let data = [];
+    // Because of getTradesV2()git  is a generator function you can loop over on the
+    // result with for/await/of
+    for await (const t of trades) {
+      data.push(t);
+    }
+    console.log("Trades for TMF");
+    console.table(data.slice(0, 5));
+
+    // Do the same for quotes as well but set a limit
+    symbol = "IBM";
+    const quotes = alpaca.getQuotesV2(symbol, { start, end });
+    data = [];
```

Comment:
This comment mentions setting a limit but it doesn't look like you've actually set one here? :thinking: